Katas/csharp/Program.cs

```diff
@@ -10,41 +10,49 @@ static void Main(string[] args)
             do
             {
                 KataAnswers.GetInputClass getInput = new KataAnswers.GetInputClass();
-                string option = getInput.GetInput("Type in your options for Katas(1-9), 0 to exit: ");
-                Console.Clear();
-                switch (option)
+                string getUserInput = getInput.GetInput("Type in your options for Katas(1-9), 0 to exit: ");
+                int option;
+                if (!int.TryParse(getUserInput, out option))
```

Comment:
`getUserInput` is a verb.+n. combination. Sounds like this is an action someone needs to do. 

The result from `getInput.GetInput` is the selected option. Thus, it could be `selectedOption` or `decision` or you name it. 

On the other hand, you can define a variable directly from `out`.

```suggestion
                string selectedOption = getInput.GetInput("Type in your options for Katas(1-9), 0 to exit: ");
                if (!int.TryParse(selectedOption, out int option))
```